src/db/spanner/manager/session.rs

```diff
@@ -23,6 +23,8 @@ pub struct SpannerSession {
     /// The underlying client (Connection/Channel) for interacting with spanner
     pub client: SpannerClient,
     pub(in crate::db::spanner) use_test_transactions: bool,
+    pub(in crate::db::spanner) create_time: i64,
+    pub(in crate::db::spanner) last_used_time: i64,
```

Comment:
As we discussed earlier, nothing updates this last_used_time. We either need to track it ourselves (not quite sure that's going to be easy) or checking the resulted version from get_session in recycle -- and also updating the value on conn there too (so the `ConnectionInfo` used for sentry extras is accurate).

Reply:
going to push back on modifying the `approximate_last_used` time for a few reasons. 

1) altering the last_used_time HEAVILY changes code in order to pass a `mut` reference down to the `async_exec()` function, where we actually use it. (We might be able to get away with an `ARC` wrapper for that, but again, things can get ugly faster than expected.
2) we have to presume that Protobuf actually works. It promises to update that whenever there's a transaction on that session, which relates to what we're interested in tracking. 
3) we can drop our `last_used_time` since it's not going to be as accurate. 

As it stands, we're reporting three values via `connection_info`:
* Our creation timestamp
* Protobuf's session creation timestamp
* Protobuf's session approximate last use timestamp.

While there may be some clock drift, it should be consistent and we can probably adjust. Plus, these are not super fine grained timers, they're seconds, some drift should be acceptable since we're monitoring on a per-minute basis.